packages/SwingSet/test/test-id.js

```diff
@@ -0,0 +1,57 @@
+import { test } from '../tools/prepare-test-env-ava.js';
+import {
+  insistVatID,
+  makeVatID,
+  insistDeviceID,
+  makeDeviceID,
+} from '../src/lib/id.js';
+
+test('makeVatID', async t => {
+  t.is(makeVatID(0), 'v0');
+  t.is(makeVatID(100n ** 10n), 'v100000000000000000000');
+
+  t.throws(() => makeVatID());
+  t.throws(() => makeVatID(-1));
+  t.throws(() => makeVatID(3.14));
+  t.throws(() => makeVatID('3'));
+  t.throws(() => makeVatID('3n'));
+});
+
+test('insistVatId', async t => {
+  t.notThrows(() => insistVatID('v0'));
+  t.notThrows(() => insistVatID('v0001'));
```

Comment:
Yeah that's the only thing that was bothering me a little.

Since we don't have any 0 id vats/devices, and we may not want to allow 0 prefixed ids, maybe the following regex would work instead?

```
/^v[1-9]\d*$`
```

Reply:
Yes, this was bothering to me as well. This test was passing for old code as well thats why I added here to be intentional and have other eyes on it. 

t.notThrows(() => insistVatID('v0001'));